openrtx/src/ui/default/ui_main.c

```diff
@@ -127,14 +127,24 @@ void _ui_drawModeInfo(ui_state_t* ui_state)
         case OPMODE_M17:
         {
             // Print M17 Destination ID on line 3 of 3
-            const char *dst = NULL;
-            if(ui_state->edit_mode)
-                dst = ui_state->new_callsign;
-            else
-                dst = (!strnlen(cfg.destination_address, 10)) ?
-                    currentLanguage->broadcast : cfg.destination_address;
+            rtxStatus_t rtxStatus = rtx_getCurrentStatus();
+            if (rtxStatus.M17_rx && rtxStatus.lsfOk) {
             gfx_print(layout.line2_pos, layout.line2_font, TEXT_ALIGN_CENTER,
-                  color_white, "#%s", dst);
+                      color_white, "dst:%s", rtxStatus.lsf_dst);
```

Comment:
Does it make sense to treat dst and src as localizable strings? If so, please add them to `stringsTable_t` and `englishStrings`.

Reply:
Probably yes.
At the moment I try to add support to the M17 code base to get the extended call sign
With it we would need to display up to 3 "call signs" for the source in addition to the destination.
Because we do not have enough space on the screen, I think using icons will be better in order to not make the screen to crammed.

Reply:
We now use icons instead  of text